server/interfaces/interfaces.go

```diff
@@ -1206,6 +1206,7 @@ type ConfigSecretProvider interface {
 
 type AuditLogger interface {
 	Log(ctx context.Context, resource *alpb.ResourceID, action alpb.Action, request proto.Message)
+	LogWithResourceType(ctx context.Context, resourceType alpb.ResourceType, id string, action alpb.Action, request proto.Message)
```

Comment:
If we're going to add a convenience method on the interface, we might as well go all the way and add type-specific functions, ie. `LogForGroup`, `LogForInvocation`, etc

Reply:
This is done.

Reply:
@vadimberezniker PTAL 🙏